packages/block-library/src/navigation/edit/index.js

```diff
@@ -488,6 +463,9 @@ function Navigation( {
 											onClose();
 										} }
 										onCreateNew={ startWithEmptyMenu }
+										canUserCreateNavigation={
```

Comment:
`showCreate` ? The component doesn't have to "know" the reason for disabling the feature.

Reply:
Fixed.